src/rhai_engine.rs

```diff
@@ -6,12 +6,19 @@ pub fn custom_rhai_engine_init() -> Engine {
     let mut rhai_engine = Engine::new();
 
     rhai_engine.register_fn("current_date", current_date);
+    rhai_engine.register_fn("print_line", print_line);
```

Comment:
Given the usage I wonder if some wording involving `log` / `debug` / `console` might clarify that this goes to the console rather than to the page itself.  My first thought was `log_debug` but that might be too prescriptive - `log_console` or `console_println` maybe?

Reply:
I like the console_println name. I'll update the PR.